tests/unit/processes/file/test_fulfill_manifest_process.py

```diff
@@ -24,7 +24,7 @@ def __init__(self):
                 self.batchSize = mocker.MagicMock(batchSize='testBatchSize')
                 self.host = mocker.MagicMock(host='testHost')
                 self.port = mocker.MagicMock(port='testPort')
-                self.prefix = 'testPrefix'
+                self.prefix = 'test_prefix'
```

Comment:
Actual variable values don't necessarily need to be in snake case!

Reply:
Oh ok thanks for the clarification. I'll change it back to camel case just to be consistent with the rest of the variable values in the test.